pshtt.py

```diff
@@ -481,6 +499,16 @@ def is_redirect(domain):
         ))
 
 
+# If a domain is a "redirect domain", where does it redirect to?
+def redirects_to(domain):
+    canonical = domain.canonical
+
+    if is_redirect(domain):
+        return canonical.redirect_eventually_to
+    else:
+        return None
```

Comment:
The `Redirect To` field is populated by whatever eventual redirect destination the domain's canonical endpoint uses. This is meant to surface what a "redirect domain" is intended to redirect users to.